lib/pages/app_grid/app_grid_dashboard_view.dart

```diff
@@ -19,71 +22,126 @@ class AppGridDashboardView extends StatelessWidget {
-                    portalFollower: const SizedBox(
-                      width: AppGridDashboardViewStyle.sizIcAppGrid,
-                    ),
-                    visible: isOpenAppGridDashboard,
-                    child: PortalTarget(
-                      anchor: const Aligned(
-                        follower: Alignment.topRight,
-                        target: Alignment.bottomRight,
-                      ),
-                      portalFollower: ValueListenableBuilder(
-                        valueListenable: controller.linagoraApplications,
-                        builder: (context, linagoraApplications, child) {
-                          if (linagoraApplications == null) return child!;
-                          return AppGridDashboardOverlay(
-                            linagoraApplications,
-                          );
-                        },
-                        child: const SizedBox.shrink(),
-                      ),
-                      visible: isOpenAppGridDashboard,
-                      child: Container(
-                        decoration: BoxDecoration(
-                          color: color,
-                          borderRadius:
-                              AppGridDashboardViewStyle.appGridIconBorderRadius,
-                        ),
-                        padding: AppGridDashboardViewStyle.appGridIconPadding,
-                        child: SvgPicture.asset(
-                          ImagePaths.icApplicationGrid,
-                          width: AppGridDashboardViewStyle.sizIcAppGrid,
-                          height: AppGridDashboardViewStyle.sizIcAppGrid,
+              onTapOutside: (_) {
+                controller.hoverColorAppHelp.value = Colors.transparent;
+              },
+              child: MouseRegion(
+                cursor: SystemMouseCursors.click,
+                onEnter: (_) => controller.hoverColorAppHelp.value =
+                    AppGridDashboardViewStyle.hoverColor,
+                onExit: (_) {
+                  controller.hoverColorAppHelp.value = Colors.transparent;
+                },
+                child: ValueListenableBuilder(
+                  valueListenable: controller.hoverColorAppHelp,
+                  builder: (context, color, _) {
+                    return TooltipVisibility(
+                      visible: true,
+                      child: Tooltip(
+                        message: L10n.of(context)!.help,
+                        showDuration: const Duration(seconds: 1),
+                        waitDuration: const Duration(seconds: 1),
+                        child: Container(
+                          decoration: BoxDecoration(
+                            color: color,
+                            borderRadius: AppGridDashboardViewStyle
+                                .appGridIconBorderRadius,
+                          ),
+                          padding: AppGridDashboardViewStyle.appGridIconPadding,
```

Comment:
it makes our icon so small.

<img width="184" alt="image" src="https://github.com/user-attachments/assets/11c0eaa4-fa89-4a81-8293-4123a20fbd41" />

Reply:
IMO: My fault when exporting the icon included background color was `White`, I updated it. pls verify it again.

Reply:
https://github.com/user-attachments/assets/cb6d7516-0da8-4d3f-b6cc-9e23ad1517fe